profiles/ding2/modules/ting_covers/ting_covers.theme.inc

```diff
@@ -42,6 +42,7 @@ function template_preprocess_ting_object_cover(&$variables) {
                               'style_name' => $variables['image_style'],
                               'path' => $path,
                               'alt' => $alt,
+                              'original_path' => $path
```

Comment:
This is need as the theme function overrides the path variable in the image style formatter and we need it at the image formatter.